### DISCUSSION TITLE
Starting Raspberry Pi Hotspot Automatically After Boot

### DISCUSSION THREAD

**Discussion by gewhjhfkgad** [RaspAP/raspap-webgui#1702] [General]
Hi all,

I am using the Raspberry Pi as a hotspot. It needs to be restarted multiple times due to changing locations. Now I am looking for a solution to start the access point (AP) automatically after the Pi boots.

Is there an easy solution for this?

Thanks!

**Comment by billz** [comment]
By default, the services required for the AP are already configured to start when the system boots. 

The simplest way to confirm this yourself is by generating a debug log and checking the "enabled" status of hostapd, dhcpcd and dnsmasq.
